[IDB IGNORE] [MDB IGNORE] Baymerge 2025

Glloyd opened the pull request:
Kill me.

This is a **very unfinished** baymerge up to mid February, but will eventually get us current with Bay once I’m finished. There’s about 2 years of PRs here, so it’s another big one, but the workload won’t be as bad as the last two because Baydev has slowed down a lot in recent years, and also we’ve cherrypicked a bunch of things from Bay in those 2 years + one of the big features (radial menus + tooltips) was a port that I did years ago, and Bay mostly followed the same file structure as me.

Please god help me with this @UristMcStation/collaborators and anyone else. I just don’t have the time to go through all of this alone. Mainly, I need someone to figure out what the hell needs to be done to Uristcode for 515/516 compatibility while I finish merging us up to present Bay, but any other help would be greatly appreciated.

Big things that we need to do:

~~https://github.com/Baystation12/Baystation12/pull/33877 - bay split the hell out of the icon folder, a bunch of urist things will probably need their icon paths fixed.~~ - mostly done, 

~~https://github.com/Baystation12/Baystation12/pull/34221/ - /obj/effects got their paths totally fucked with, a lot of things will need their paths changed~~ - should be done, but worth keeping an eye on

Bay has updated to 515, and is in the process of updating to 516 (I’m working on doing another merge to this PR to get us fully up to date with current bay). We will need to update Uristcode to 515, which will be a hassle, and also 516, which is less of a hassle.

Other TODO list, probably after the above is dealt with:
Bay reworked their skill system and made skills even more important, so new skill checks were probably added and need to be purged or adjusted. Existing ones that we had already removed have already been dealt with.

- [x] Make it compile (will need 515 compat for this)
- [x] Fix the maps (mainly will be repathing /obj/effects, i'll take care of this for glloydstation and nerva)
- [x] Fix the tests (kill me)
- [ ] Purge update_operator_skill
- [ ] restore Urist holopad functionality
- [ ] new paths will need to be added to @CPU-Blanc 's generate worths list

Things that probably need testing based on some tricky conflict resolving or other issues:
check holopads functionality
check radial/tooltip functionality – surgery should be fine, I’ll have to check on ladders. tooltips in particular might have some funky overlap, Bay’s implementation was slightly different than mine.
check building mechs
check TEG anchoring
scrdest’s zombie changes may have been partially lost
~~Some of our changes to bay aways were lost, but they were mostly minor changes, I think I preserved most of the big ones.~~
~~The moving_quickly thing in mob_movement.dm. What was changed here in the last merge @ggermanicus?~~
Floating messages – Bay also ported floating messages, so there might be some minor issues here, probably formatting but we’ll need to see once we get this running.
Antag species restrictions might have gotten  bit fucked up, investigate this.
Check post_fire for weapons to make sure that the dual weapon check still works properly
Shotguns may have been messed with, will need a quick pass
We are now using Bay’s human sprite instead of our old one. We can decide which we prefer.
new paths will need to be added to @CPU-Blanc 's generate worths list
Check hug proc to make sure it works properly

Rough, very incomplete changelog:

515+ compatibility!!!
New lighting system, including emissives
New changeling mode
New bunch of new items/clothes/foods/reciples/etc. and some new sprites too
Bay added their own weather system I’ll need to look into
A bunch of minor features that I can’t be bothered listing, including some things we already cherry-picked (bulletholes in walls, ninja rework, pronouns, etc.)
Some qol things, including some nice qol for loading magazines.
Bugfixes, optimizations, and a whole bunch of pointless refactors.

CrimsonShrike committed:
Ports weather from nebula

CrimsonShrike committed:
ZAS and atmos changes, planets and visitable overmap locations now can define external atmos

HeyBanditoz committed:
Port Nebula's airlock helpers

With adjustments to fix Bay airlocks.
As an alternative for mapping airlocks.

Detritus-Ha committed:
The kind of fancy extractor

Added and loaded up the extractor

+ tiny grammar checks
+ nanoui

Co-Authored-By: Hayden <banditoz@protonmail.com>

Spookerton committed:
Merge pull request #35193 from Spookerton/spkrtn/cng/lazy-housekeeping

lazy housekeeping

Spookerton committed:
fix area copy to area using source coords instead of target for target turfs

Spookerton committed:
Merge pull request #35191 from CakeQ/bloody-hugs-fix

Fixes blood transferral on hugs

Bay12Bot committed:
Automatic changelog generation for PR #35191 [ci skip]

Spookerton committed:
Merge pull request #35196 from Spookerton/spkrtn/fix/holodeck-area-copy

fix area copy to area using source coords instead of target for target turfs

Bay12Bot committed:
Automatic changelog generation for PR #35196 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Detritus-Ha committed:
All unary machines now return_air

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Detritus-Ha committed:
Binary devices now return air

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Expand the playable colony's mess and medbay

* Expands the mess hall and kitchen area.
* The lounge "common area" was removed to make room for kitchen space.
* The medbay is now one tile wider.
* Adds a fridge and organ printer to the medbay.

Bay12Bot committed:
Automatic changelog generation [ci skip]

Ryan180602 committed:
doubles name input on serving bowl food

MuckerMayhem committed:
Merge pull request #35147 from CakeQ/bootflask

Flasks can be hidden in boots

MuckerMayhem committed:
Merge pull request #35203 from Ryan180602/food-unbound

Increase name limit on pen -> food / Doubles name limit on serving bowl

Bay12Bot committed:
Automatic changelog generation for PR #35147 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35203 [ci skip]

Spookerton committed:
Merge pull request #35179 from HeyBanditoz/pda_health_beep_beep

Medical scanner modules now beep on scan (PDAs, tablets)

Bay12Bot committed:
Automatic changelog generation for PR #35179 [ci skip]

Spookerton committed:
Merge pull request #35134 from HeyBanditoz/mutation_body_scanning

Upgraded body scanners check for mutation existence

Bay12Bot committed:
Automatic changelog generation for PR #35134 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Glloyd committed:
fixes 1605

Glloyd committed:
Merge pull request #1606 from Glloyd/let-the-tesh-blood-flow

Fixes #1605

MuckerMayhem committed:
Merge pull request #35197 from Detritus-Ha/analyze

All unary machines are scannable

MuckerMayhem committed:
Merge pull request #35200 from Detritus-Ha/analyze-binary

All binary machines are scannable

Bay12Bot committed:
Automatic changelog generation for PR #35197 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35200 [ci skip]

MuckerMayhem committed:
Merge pull request #35120 from CrimsonShrike/WeatherAndPlanets

Weather and planet Air

Bay12Bot committed:
Automatic changelog generation for PR #35120 [ci skip]

CrimsonShrike committed:
Ports blurbs from nebula, modifies them a bit

Bay12Bot committed:
Automatic changelog generation [ci skip]

CrimsonShrike committed:
rain sound fixes

MuckerMayhem committed:
actually fix blood hugs

Spookerton committed:
Merge pull request #35206 from MuckerMayhem/really-fix-bloodhugs

actually fix blood hugs

Bay12Bot committed:
Automatic changelog generation for PR #35206 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
Merge pull request #35171 from Detritus-Ha/extraction-v4

Adds the Gas Extractor

MuckerMayhem committed:
Merge pull request #35205 from CrimsonShrike/HaveYouEverHeardTheRain

rain sound fixes

Bay12Bot committed:
Automatic changelog generation for PR #35171 [ci skip]

MuckerMayhem committed:
Merge pull request #35143 from HeyBanditoz/airlock_helpers

Port Nebula's airlock helpers

Bay12Bot committed:
Automatic changelog generation for PR #35143 [ci skip]

MuckerMayhem committed:
Merge pull request #35204 from CrimsonShrike/RibbonFeatures

Ports blurbs from nebula, modifies them a bit

Bay12Bot committed:
Automatic changelog generation [ci skip]

RuneByt3 committed:
prevents robots from renegade

RuneByt3 committed:
GAS no longer Cough

RuneByt3 committed:
fixes hardsuit module irregularities

MuckerMayhem committed:
Merge pull request #35202 from HeyBanditoz/playable_colony_mess_medbay_expansion

Expand the playable colony's mess and medbay

Bay12Bot committed:
Automatic changelog generation for PR #35202 [ci skip]

MuckerMayhem committed:
Merge pull request #35207 from RuneByt3/bugfix

prevents robots from renegade

Bay12Bot committed:
Automatic changelog generation for PR #35207 [ci skip]

Fre3bie committed:
Adds extra check for equip slot

RuneByt3 committed:
No monkey pick up with backslot item

Sierra code suggestion

Co-authored-by: SierraKomodo <sierrakomodo@hey.com>

fix bracket

Bay12Bot committed:
Automatic changelog generation [ci skip]

Detritus-Ha committed:
Better injector

dependabot[bot] committed:
Bump actions/cache from 4.2.2 to 4.2.3

Bumps [actions/cache](https://github.com/actions/cache) from 4.2.2 to 4.2.3.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/d4323d4df104b026a6aa633fdb11d772146be0bf...5a3ec84eff668545956fd18022155c47e93e2684)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
Merge pull request #35213 from Baystation12/dependabot/github_actions/actions/cache-4.2.3

Bump actions/cache from 4.2.2 to 4.2.3

Laniakeaaa committed:
Additional Glloyd maint

stellarzenith committed:
Fix hugs misgendering you

Glloyd committed:
Merge pull request #1609 from Laniakeaaa/Glloydstation_Changes

Glloydstation - Additional Spring Cleaning

Laniakeaaa committed:
Fixing Pass

ggermanicus committed:
Fixes unwanted IPC marking sprites

Spookerton committed:
Merge pull request #35065 from CakeQ/multispawn-prefs

Adds the ability to ready up as multiple characters

Bay12Bot committed:
Automatic changelog generation for PR #35065 [ci skip]

Spookerton committed:
Merge pull request #35208 from RuneByt3/GASCough

GAS no longer Cough

Bay12Bot committed:
Automatic changelog generation for PR #35208 [ci skip]

Spookerton committed:
Merge pull request #35210 from Fre3bie/glove-fix

Adds extra check for equip slot

Bay12Bot committed:
Automatic changelog generation for PR #35210 [ci skip]

Spookerton committed:
Merge pull request #35211 from RuneByt3/monkeybusiness

Preventing pick up monkey with back item

Spookerton committed:
Merge pull request #35214 from stellarzenith/make-hugs-woke

Fix hugs misgendering you

Bay12Bot committed:
Automatic changelog generation for PR #35214 [ci skip]

Spookerton committed:
Merge pull request #35186 from BurpleBineapple/armblades

Corporate Suit Dash Tweaks (again)

Bay12Bot committed:
Automatic changelog generation for PR #35186 [ci skip]

Spookerton committed:
add test run duration to test outputs

Spookerton committed:
use find instead of in for fusion_network_test vars check

Spookerton committed:
Merge pull request #35215 from Spookerton/spkrtn/sys/fix-fusion-test-slow

fix fusion_network_test performance

dependabot[bot] committed:
Bump actions/setup-python from 5.4.0 to 5.5.0

Bumps [actions/setup-python](https://github.com/actions/setup-python) from 5.4.0 to 5.5.0.
- [Release notes](https://github.com/actions/setup-python/releases)
- [Commits](https://github.com/actions/setup-python/compare/42375524e23c412d93fb67b49958b491fce71c38...8d9ed9ac5c53483de85588cdf95a591a75ab9f55)

---
updated-dependencies:
- dependency-name: actions/setup-python
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
Merge pull request #35216 from Baystation12/dependabot/github_actions/actions/setup-python-5.5.0

Bump actions/setup-python from 5.4.0 to 5.5.0

MuckerMayhem committed:
Merge pull request #35212 from Detritus-Ha/betterinjectors

Cleaner injectors

Bay12Bot committed:
Automatic changelog generation for PR #35212 [ci skip]

Spookerton committed:
Merge pull request #35209 from RuneByt3/hardsuitfixes

hardsuit thrusters don't work for floored/dead people

Bay12Bot committed:
Automatic changelog generation for PR #35209 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Glloyd committed:
Merge pull request #1614 from ggermanicus/cosmeticfix

IPC body markings no longer add random face pixels

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

thatdanguy23 committed:
add three dresses, collect two dress gear selections into one

also cleans up dress pathing a little

Laniakeaaa committed:
Adjusting Desc, Names, Moving, etc.

Laniakeaaa committed:
Holdout Revolver & Pistol Bullets fit in shoes.

Laniakeaaa committed:
Ports Bay one, too.

Laniakeaaa committed:
Dartgun UX

Spookerton committed:
Merge pull request #35217 from thatdanguy23/casual-dresses

Casual dresses & loadout

Bay12Bot committed:
Automatic changelog generation for PR #35217 [ci skip]

ggermanicus committed:
broadcasts available offship roles roundstart

cosmoslego committed:
Fix teleporter not working for mobs in closet

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
limit overmap spawns from being on the map limits

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
Merge pull request #35219 from MuckerMayhem/overmap-event-tweak

limit overmap spawns from being on the map limits

Bay12Bot committed:
Automatic changelog generation for PR #35219 [ci skip]

Spookerton committed:
Merge pull request #35218 from korolKlinkoff/dev

Fix teleporter not working for mobs in closet

Bay12Bot committed:
Automatic changelog generation for PR #35218 [ci skip]

Glloyd committed:
Merge pull request #1617 from Laniakeaaa/MoreFixingTraitorShit

Traitor Uplink - Cleaning & Maintenance

Glloyd committed:
Merge pull request #1619 from Laniakeaaa/DartGunUI

Dartguns now have Action Buttons for Mixing Controls.

Glloyd committed:
Merge pull request #1618 from Laniakeaaa/StealthPass

Holdout Revolvers, Single Pistol Bullets & Screwdrivers can be hidden in shoes.

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
HTML encode destination tags

Fixes destinations with custom characters in their destination name, like `Pathfinder's Office`

HeyBanditoz committed:
Remove welding fuel icon from roundstart blurb

ggermanicus committed:
replaces all urist bandoliers with bay ones

Glloyd committed:
Merge pull request #1621 from ggermanicus/retirementhome

replaces all urist bandoliers with bay ones

Bay12Bot committed:
Automatic changelog generation [ci skip]

ggermanicus committed:
fix mark-and-recall and create portal

ggermanicus committed:
141 to_worlds

Glloyd committed:
Merge pull request #1622 from ggermanicus/wiz

fix mark-and-recall and create portal

Glloyd committed:
Merge pull request #1623 from ggermanicus/rolesavailable

game broadcasts available offship roles at roundstart

ggermanicus committed:
ports bay changes to ions

Laniakeaaa committed:
Lar Maria

Glloyd committed:
Merge pull request #1625 from Laniakeaaa/AwayAdjustments

Lar Maria - Away Map Update

Glloyd committed:
Merge pull request #1624 from ggermanicus/emping-limbs-2

ports bay changes to ions

Laniakeaaa committed:
Mutineer Icon Fix

MuckerMayhem committed:
Merge pull request #35225 from HeyBanditoz/no_more_welding_fuel

Remove welding fuel icon from roundstart blurb

MuckerMayhem committed:
Merge pull request #35224 from HeyBanditoz/destination_tag_fix

HTML encode destination tags

Bay12Bot committed:
Automatic changelog generation for PR #35224 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

ggermanicus committed:
removes unintended interactions opening arms safes

Laniakeaaa committed:
gantry ships can now finally spawn

Glloyd committed:
Merge pull request #1628 from Laniakeaaa/gantryfix

Pirate Gantry's can finally spawn.

Glloyd committed:
Merge pull request #1627 from ggermanicus/no-fun-allowed

removes unintended interactions opening arms safes

Glloyd committed:
Merge pull request #1626 from Laniakeaaa/hudfix

Fixes Mutineer/Raider HUD Icon defaulting to error HUD icon.

Bay12Bot committed:
Automatic changelog generation [ci skip]

Laniakeaaa committed:
Adds a balaclava

Laniakeaaa committed:
binary key desc proper + tc cost

Laniakeaaa committed:
Verbage, Fixes Polydye bug, naming, etc.

Laniakeaaa committed:
Adds Maneater Plants, Moonshine, Goonstation Mob + Credit.

Bay12Bot committed:
Automatic changelog generation [ci skip]

Laniakeaaa committed:
monkeys have a chance to activate grenades now

Bay12Bot committed:
Automatic changelog generation [ci skip]

Glloyd committed:
Merge branch 'dev' of https://github.com/Baystation12/Baystation12 into baymerge-testing

# Conflicts:
#	.github/labeler.yml
#	.github/workflows/changelog_generation.yml
#	.github/workflows/close_stale.yml
#	.github/workflows/codeql-analysis.yml
#	.github/workflows/generate_documentation.yml
#	.github/workflows/make_changelogs.yml
#	.github/workflows/test.yml
#	README.md
#	baystation12.dme
#	code/__defines/__renderer.dm
#	code/__defines/culture.dm
#	code/__defines/languages.dm
#	code/__defines/machinery.dm
#	code/__defines/mobs.dm
#	code/__defines/time.dm
#	code/_global_vars/lists/flavor.dm
#	code/_global_vars/misc.dm
#	code/_helpers/cmp.dm
#	code/_helpers/maths.dm
#	code/_helpers/medical_scans.dm
#	code/_helpers/mobs.dm
#	code/_helpers/text.dm
#	code/_helpers/time.dm
#	code/_onclick/ai.dm
#	code/_onclick/hud/radial.dm
#	code/_onclick/hud/radial_persistent.dm
#	code/_version.dm
#	code/controllers/configuration.dm
#	code/controllers/hooks-defs.dm
#	code/controllers/master.dm
#	code/controllers/subsystems/ai.dm
#	code/controllers/subsystems/aifast.dm
#	code/controllers/subsystems/mobs.dm
#	code/controllers/subsystems/skybox.dm
#	code/controllers/subsystems/supply.dm
#	code/controllers/subsystems/ticker.dm
#	code/controllers/subsystems/timer.dm
#	code/core/math/math.dm
#	code/datums/extensions/holster/holster.dm
#	code/datums/mind/mind.dm
#	code/datums/movement/mob.dm
#	code/datums/supplypacks/engineering.dm
#	code/datums/supplypacks/operations.dm
#	code/datums/trading/trade.dm
#	code/datums/uplink/ammunition.dm
#	code/datums/uplink/devices and tools.dm
#	code/datums/uplink/highly_visible_and_dangerous_weapons.dm
#	code/datums/uplink/stealth_and_camouflage_items.dm
#	code/datums/wires/airlock.dm
#	code/datums/wires/alarm.dm
#	code/datums/wires/apc.dm
#	code/datums/wires/camera.dm
#	code/datums/wires/fabricator.dm
#	code/datums/wires/nuclearbomb.dm
#	code/datums/wires/particle_accelerator.dm
#	code/datums/wires/radio.dm
#	code/datums/wires/robot.dm
#	code/datums/wires/shield_generator.dm
#	code/datums/wires/smartfridge.dm
#	code/datums/wires/smes.dm
#	code/datums/wires/suit_storage_unit.dm
#	code/datums/wires/taperecorder.dm
#	code/datums/wires/wire_description.dm
#	code/datums/wires/wires.dm
#	code/game/antagonist/alien/borer.dm
#	code/game/antagonist/antagonist_print.dm
#	code/game/antagonist/outsider/ert.dm
#	code/game/antagonist/outsider/ninja.dm
#	code/game/antagonist/outsider/raider.dm
#	code/game/antagonist/outsider/vox.dm
#	code/game/antagonist/outsider/wizard.dm
#	code/game/antagonist/station/changeling.dm
#	code/game/antagonist/station/cultist.dm
#	code/game/antagonist/station/revolutionary.dm
#	code/game/antagonist/station/traitor.dm
#	code/game/area/areas.dm
#	code/game/atoms.dm
#	code/game/atoms_health.dm
#	code/game/gamemodes/changeling/absorbed_dna.dm
#	code/game/gamemodes/changeling/changeling.dm
#	code/game/gamemodes/changeling/changeling_powers.dm
#	code/game/gamemodes/endgame/bluespace_jump/bluespace_jump.dm
#	code/game/gamemodes/malfunction/newmalf_ability_trees/tree_networking.dm
#	code/game/gamemodes/meteor/meteors.dm
#	code/game/gamemodes/ninja/ninja.dm
#	code/game/machinery/Sleeper.dm
#	code/game/machinery/alarm.dm
#	code/game/machinery/bluespace_drive.dm
#	code/game/machinery/computer/Operating.dm
#	code/game/machinery/cryopod.dm
#	code/game/machinery/doors/airlock.dm
#	code/game/machinery/doors/door.dm
#	code/game/machinery/doors/multi_tile.dm
#	code/game/machinery/hologram.dm
#	code/game/machinery/holosign.dm
#	code/game/machinery/jukebox.dm
#	code/game/machinery/kitchen/cookers.dm
#	code/game/machinery/kitchen/smartfridge.dm
#	code/game/machinery/newscaster.dm
#	code/game/machinery/suit_cycler.dm
#	code/game/machinery/suit_cycler_units.dm
#	code/game/machinery/supplybeacon.dm
#	code/game/machinery/teleporter/beacon.dm
#	code/game/machinery/teleporter/pad.dm
#	code/game/machinery/vending/_wires.dm
#	code/game/machinery/vending/cigarette.dm
#	code/game/machinery/vending/games.dm
#	code/game/machinery/vending/snack.dm
#	code/game/machinery/vitals_monitor.dm
#	code/game/objects/buckling.dm
#	code/game/objects/effects/decals/Cleanable/humans.dm
#	code/game/objects/effects/decals/Cleanable/misc.dm
#	code/game/objects/effects/decals/warning_stripes.dm
#	code/game/objects/effects/landmarks.dm
#	code/game/objects/effects/mines.dm
#	code/game/objects/effects/misc.dm
#	code/game/objects/effects/particles/particles.dm
#	code/game/objects/items.dm
#	code/game/objects/items/devices/auto_cpr.dm
#	code/game/objects/items/devices/flashlight.dm
#	code/game/objects/items/devices/inducer.dm
#	code/game/objects/items/devices/personal_shield.dm
#	code/game/objects/items/devices/radio/headset.dm
#	code/game/objects/items/devices/radio/radio.dm
#	code/game/objects/items/devices/taperecorder.dm
#	code/game/objects/items/devices/uplink_random_lists.dm
#	code/game/objects/items/gunboxes.dm
#	code/game/objects/items/stacks/medical.dm
#	code/game/objects/items/stacks/stack.dm
#	code/game/objects/items/weapons/AI_modules.dm
#	code/game/objects/items/weapons/RPD.dm
#	code/game/objects/items/weapons/candle/candle.dm
#	code/game/objects/items/weapons/cards_ids.dm
#	code/game/objects/items/weapons/circuitboards/circuitboard.dm
#	code/game/objects/items/weapons/circuitboards/computer/computer.dm
#	code/game/objects/items/weapons/circuitboards/machinery/mining_drill.dm
#	code/game/objects/items/weapons/implants/implantpad.dm
#	code/game/objects/items/weapons/implants/implants/explosive.dm
#	code/game/objects/items/weapons/lighter.dm
#	code/game/objects/items/weapons/material/coins.dm
#	code/game/objects/items/weapons/material/kitchen.dm
#	code/game/objects/items/weapons/melee/energy.dm
#	code/game/objects/items/weapons/policetape.dm
#	code/game/objects/items/weapons/storage/bags.dm
#	code/game/objects/items/weapons/storage/boxes.dm
#	code/game/objects/items/weapons/storage/lockbox.dm
#	code/game/objects/items/weapons/storage/med_pouch.dm
#	code/game/objects/items/weapons/storage/misc.dm
#	code/game/objects/items/weapons/storage/secure.dm
#	code/game/objects/items/weapons/tanks/tanks.dm
#	code/game/objects/items/weapons/tape.dm
#	code/game/objects/items/weapons/trays.dm
#	code/game/objects/random/random.dm
#	code/game/objects/structures/coathanger.dm
#	code/game/objects/structures/crates_lockers/closets/__closet.dm
#	code/game/objects/structures/crates_lockers/closets/secure/hydroponics.dm
#	code/game/objects/structures/fireaxe_cabinet.dm
#	code/game/objects/structures/fitness.dm
#	code/game/objects/structures/janicart.dm
#	code/game/objects/structures/plasticflaps.dm
#	code/game/objects/structures/stool_bed_chair_nest/bed.dm
#	code/game/objects/structures/under_wardrobe.dm
#	code/game/objects/structures/wall_frame.dm
#	code/game/objects/structures/wallframe_spawner.dm
#	code/game/objects/structures/watercloset.dm
#	code/game/turfs/flooring/flooring.dm
#	code/game/turfs/flooring/flooring_decals.dm
#	code/game/turfs/flooring/flooring_premade.dm
#	code/game/turfs/simulated/floor_attackby.dm
#	code/game/turfs/simulated/wall_attacks.dm
#	code/game/turfs/simulated/walls.dm
#	code/game/world.dm
#	code/modules/admin/admin.dm
#	code/modules/admin/callproc/callproc.dm
#	code/modules/admin/verbs/adminhelp.dm
#	code/modules/admin/verbs/adminpm.dm
#	code/modules/ai/ai_holder_movement.dm
#	code/modules/ai/ai_holder_targeting.dm
#	code/modules/atmospherics/components/binary_devices/circulator.dm
#	code/modules/augment/active/armblades.dm
#	code/modules/augment/active/iatric_monitor.dm
#	code/modules/augment/augment.dm
#	code/modules/augment/implanter.dm
#	code/modules/blob/blob.dm
#	code/modules/client/client_color.dm
#	code/modules/client/client_defines.dm
#	code/modules/client/client_procs.dm
#	code/modules/client/preference_setup/antagonism/01_candidacy.dm
#	code/modules/client/preference_setup/general/02_body.dm
#	code/modules/client/preference_setup/global/preferences.dm
#	code/modules/client/preference_setup/loadout/lists/misc.dm
#	code/modules/client/preference_setup/loadout/lists/tactical.dm
#	code/modules/client/preference_setup/loadout/lists/xenowear.dm
#	code/modules/client/preference_setup/loadout/loadout.dm
#	code/modules/clothing/chameleon.dm
#	code/modules/clothing/head/helmet.dm
#	code/modules/clothing/head/jobs.dm
#	code/modules/clothing/shoes/miscellaneous.dm
#	code/modules/clothing/spacesuits/rig/modules/combat.dm
#	code/modules/clothing/spacesuits/rig/rig_attackby.dm
#	code/modules/clothing/spacesuits/rig/rig_pieces.dm
#	code/modules/clothing/spacesuits/rig/rig_wiring.dm
#	code/modules/clothing/spacesuits/rig/suits/light.dm
#	code/modules/clothing/spacesuits/rig/suits/station.dm
#	code/modules/clothing/spacesuits/spacesuits.dm
#	code/modules/clothing/suits/miscellaneous.dm
#	code/modules/clothing/under/skirts.dm
#	code/modules/codex/entries/medical.dm
#	code/modules/culture_descriptor/faction/factions_human.dm
#	code/modules/detectivework/microscope/microscope.dm
#	code/modules/detectivework/tools/crimekit.dm
#	code/modules/detectivework/tools/rag.dm
#	code/modules/detectivework/tools/sample_kits.dm
#	code/modules/economy/cash.dm
#	code/modules/events/bsd_instability.dm
#	code/modules/events/event_container.dm
#	code/modules/events/meteors.dm
#	code/modules/fabrication/designs/general/designs_arms_ammo.dm
#	code/modules/fabrication/designs/general/designs_tools.dm
#	code/modules/holodeck/HolodeckObjects.dm
#	code/modules/holomap/ship_holomap.dm
#	code/modules/hydroponics/seed_datums.dm
#	code/modules/hydroponics/seed_machines.dm
#	code/modules/hydroponics/spreading/spreading.dm
#	code/modules/hydroponics/spreading/spreading_growth.dm
#	code/modules/hydroponics/trays/tray.dm
#	code/modules/integrated_electronics/core/assemblies.dm
#	code/modules/integrated_electronics/subtypes/reagents.dm
#	code/modules/maps/ruins.dm
#	code/modules/materials/material_sheets.dm
#	code/modules/mechs/components/_components.dm
#	code/modules/mechs/components/frame.dm
#	code/modules/mechs/components/software.dm
#	code/modules/mechs/equipment/utility.dm
#	code/modules/mechs/mech_damage.dm
#	code/modules/mechs/mech_interaction.dm
#	code/modules/mining/mine_turfs.dm
#	code/modules/mob/floating_message.dm
#	code/modules/mob/grab/normal/grab_normal.dm
#	code/modules/mob/living/bot/bot.dm
#	code/modules/mob/living/carbon/brain/MMI.dm
#	code/modules/mob/living/carbon/carbon_defines.dm
#	code/modules/mob/living/carbon/human/appearance.dm
#	code/modules/mob/living/carbon/human/corpse.dm
#	code/modules/mob/living/carbon/human/examine.dm
#	code/modules/mob/living/carbon/human/human.dm
#	code/modules/mob/living/carbon/human/human_attackhand.dm
#	code/modules/mob/living/carbon/human/human_damage.dm
#	code/modules/mob/living/carbon/human/human_defense.dm
#	code/modules/mob/living/carbon/human/human_movement.dm
#	code/modules/mob/living/carbon/human/human_species.dm
#	code/modules/mob/living/carbon/human/unarmed_attack.dm
#	code/modules/mob/living/living.dm
#	code/modules/mob/living/say.dm
#	code/modules/mob/living/silicon/ai/ai.dm
#	code/modules/mob/living/silicon/pai/pai.dm
#	code/modules/mob/living/silicon/posi_brainmob.dm
#	code/modules/mob/living/silicon/robot/drone/drone.dm
#	code/modules/mob/living/silicon/robot/modules/module_janitor.dm
#	code/modules/mob/living/silicon/robot/modules/module_medical.dm
#	code/modules/mob/living/silicon/robot/robot.dm
#	code/modules/mob/living/simple_animal/defense.dm
#	code/modules/mob/living/simple_animal/familiars/familiars.dm
#	code/modules/mob/living/simple_animal/friendly/farm_animals.dm
#	code/modules/mob/living/simple_animal/hostile/giant_spider/webslinger.dm
#	code/modules/mob/living/simple_animal/hostile/retaliate/parrot.dm
#	code/modules/mob/living/simple_animal/hostile/syndicate.dm
#	code/modules/mob/living/simple_animal/simple_animal.dm
#	code/modules/mob/mob.dm
#	code/modules/mob/mob_movement.dm
#	code/modules/mob/pronouns.dm
#	code/modules/mob/skills/skill.dm
#	code/modules/mob/skills/skill_verbs.dm
#	code/modules/modular_computers/computers/subtypes/dev_pda.dm
#	code/modules/modular_computers/file_system/programs/antagonist/access_decrypter.dm
#	code/modules/modular_computers/file_system/programs/antagonist/dos.dm
#	code/modules/modular_computers/file_system/programs/engineering/supermatter_monitor.dm
#	code/modules/modular_computers/file_system/programs/research/ai_restorer.dm
#	code/modules/modular_computers/file_system/reports/crew_record.dm
#	code/modules/modular_computers/terminal/commands/ban.dm
#	code/modules/modular_computers/terminal/commands/hwinfo.dm
#	code/modules/modular_computers/terminal/commands/ifconfig.dm
#	code/modules/modular_computers/terminal/commands/locate.dm
#	code/modules/modular_computers/terminal/commands/lock.dm
#	code/modules/modular_computers/terminal/commands/log.dm
#	code/modules/modular_computers/terminal/commands/netscan.dm
#	code/modules/modular_computers/terminal/commands/ping.dm
#	code/modules/modular_computers/terminal/commands/prog.dm
#	code/modules/modular_computers/terminal/commands/proxy.dm
#	code/modules/modular_computers/terminal/commands/purge.dm
#	code/modules/modular_computers/terminal/commands/relays.dm
#	code/modules/modular_computers/terminal/commands/ssh.dm
#	code/modules/modular_computers/terminal/commands/status.dm
#	code/modules/modular_computers/terminal/commands/sysnotify.dm
#	code/modules/multiz/movement.dm
#	code/modules/multiz/structures.dm
#	code/modules/organs/external/_external.dm
#	code/modules/organs/external/_external_icons.dm
#	code/modules/organs/external/diagnostics.dm
#	code/modules/organs/internal/species/fbp.dm
#	code/modules/organs/internal/species/ipc.dm
#	code/modules/organs/internal/species/vox.dm
#	code/modules/organs/internal/stomach.dm
#	code/modules/organs/organ.dm
#	code/modules/organs/robolimbs.dm
#	code/modules/overmap/disperser/disperser_console.dm
#	code/modules/overmap/exoplanets/_exoplanet.dm
#	code/modules/overmap/exoplanets/exoplanet_skybox.dm
#	code/modules/overmap/overmap_shuttle.dm
#	code/modules/overmap/sectors.dm
#	code/modules/overmap/ships/computers/sensors.dm
#	code/modules/overmap/ships/ship.dm
#	code/modules/paperwork/adminpaper.dm
#	code/modules/paperwork/filingcabinet.dm
#	code/modules/paperwork/paper.dm
#	code/modules/paperwork/pen/reagent_pen.dm
#	code/modules/power/cable_structure.dm
#	code/modules/power/fusion/core/_core.dm
#	code/modules/power/fusion/core/core_field.dm
#	code/modules/power/generator.dm
#	code/modules/power/lighting.dm
#	code/modules/power/singularity/emitter.dm
#	code/modules/power/solar.dm
#	code/modules/projectiles/ammunition.dm
#	code/modules/projectiles/ammunition/boxes.dm
#	code/modules/projectiles/effects.dm
#	code/modules/projectiles/gun.dm
#	code/modules/projectiles/guns/energy/nuclear.dm
#	code/modules/projectiles/guns/energy/special.dm
#	code/modules/projectiles/guns/launcher/crossbow.dm
#	code/modules/projectiles/guns/projectile.dm
#	code/modules/projectiles/guns/projectile/automatic.dm
#	code/modules/projectiles/guns/projectile/flaregun.dm
#	code/modules/projectiles/guns/projectile/revolver.dm
#	code/modules/projectiles/guns/projectile/shotgun.dm
#	code/modules/projectiles/guns/projectile/sniper.dm
#	code/modules/projectiles/projectile.dm
#	code/modules/projectiles/projectile/bullets.dm
#	code/modules/projectiles/projectile/energy.dm
#	code/modules/projectiles/targeting/targeting_triggers.dm
#	code/modules/reagents/Chemistry-Grinder.dm
#	code/modules/reagents/Chemistry-Reagents/Random/Chemistry-Reagents-Random.dm
#	code/modules/reagents/Chemistry-Reagents/Random/random_effects.dm
#	code/modules/reagents/Chemistry-Recipes.dm
#	code/modules/reagents/dispenser/dispenser2.dm
#	code/modules/reagents/reagent_containers.dm
#	code/modules/reagents/reagent_containers/borghypo.dm
#	code/modules/reagents/reagent_containers/food/fish.dm
#	code/modules/reagents/reagent_containers/food/sandwich.dm
#	code/modules/reagents/reagent_containers/food/shaker.dm
#	code/modules/reagents/reagent_containers/food/snacks.dm
#	code/modules/reagents/reagent_containers/food/snacks/shellfish.dm
#	code/modules/reagents/reagent_containers/hypospray.dm
#	code/modules/reagents/reagent_containers/ivbag.dm
#	code/modules/reagents/reagent_dispenser.dm
#	code/modules/recycling/sortingmachinery.dm
#	code/modules/research/protolathe.dm
#	code/modules/security levels/keycard authentication.dm
#	code/modules/shieldgen/shieldwallgen.dm
#	code/modules/species/outsider/vox.dm
#	code/modules/species/outsider/zombie.dm
#	code/modules/species/species.dm
#	code/modules/species/station/human_subspecies.dm
#	code/modules/species/station/machine.dm
#	code/modules/species/station/nabber.dm
#	code/modules/species/station/prometheans.dm
#	code/modules/species/station/station.dm
#	code/modules/spells/aoe_turf/exchange_wounds.dm
#	code/modules/spells/artifacts/spellbound_servants.dm
#	code/modules/spells/hand/hand.dm
#	code/modules/spells/hand/hand_item.dm
#	code/modules/spells/targeted/cleric_spells.dm
#	code/modules/spells/targeted/equip/seed.dm
#	code/modules/spells/targeted/ethereal_jaunt.dm
#	code/modules/spells/targeted/shapeshift.dm
#	code/modules/supermatter/supermatter.dm
#	code/modules/surgery/__surgery_setup.dm
#	code/modules/surgery/generic.dm
#	code/modules/surgery/implant.dm
#	code/modules/surgery/organs_internal.dm
#	code/modules/surgery/robotics.dm
#	code/modules/surgery/slimes.dm
#	code/modules/surgery/surgery.dm
#	code/modules/tables/tables.dm
#	code/modules/tooltip/tooltip.dm
#	code/modules/tooltip/tooltip.html
#	code/modules/xenoarcheaology/finds/misc.dm
#	code/modules/xenoarcheaology/tools/artifact_analyser.dm
#	code/modules/xenoarcheaology/triggers/chemical.dm
#	code/unit_tests/closets.dm
#	code/unit_tests/~unit_test_subsystems.dm
#	config/example/config.txt
#	icons/mob/human_races/species/human/body.dmi
#	icons/mob/human_races/species/ipc/markings.dmi
#	icons/mob/onmob/onmob_feet.dmi
#	icons/mob/onmob/onmob_hands.dmi
#	icons/mob/onmob/onmob_head.dmi
#	icons/mob/onmob/onmob_suit.dmi
#	icons/obj/alien.dmi
#	icons/obj/ammo.dmi
#	icons/obj/barsigns.dmi
#	icons/obj/closets/decals/closet.dmi
#	icons/obj/clothing/obj_feet.dmi
#	icons/obj/clothing/obj_hands.dmi
#	icons/obj/clothing/obj_head.dmi
#	icons/obj/clothing/obj_suit.dmi
#	icons/obj/cryogenics_split.dmi
#	icons/obj/food.dmi
#	icons/obj/food_custom.dmi
#	icons/obj/food_ingredients.dmi
#	icons/obj/guns/holdout_pistol.dmi
#	icons/obj/hydroponics_products.dmi
#	icons/obj/janitor.dmi
#	icons/obj/monitors.dmi
#	icons/obj/power_cond_white.dmi
#	icons/obj/rig_modules.dmi
#	icons/obj/seeds.dmi
#	icons/obj/telescience.dmi
#	icons/obj/tools/batterer.dmi
#	icons/obj/turbolift.dmi
#	icons/screen/radial.dmi
#	maps/_maps.dm
#	maps/antag_spawn/ert/ert_base.dmm
#	maps/antag_spawn/mercenary/mercenary.dm
#	maps/antag_spawn/ninja/ninja.dm
#	maps/antag_spawn/ninja/ninja_base.dmm
#	maps/antag_spawn/vox/voxraider.dm
#	maps/away/bearcat/bearcat-1.dmm
#	maps/away/bearcat/bearcat-2.dmm
#	maps/away/blueriver/blueriver-2.dmm
#	maps/away/blueriver/blueriver.dm
#	maps/away/casino/casino.dmm
#	maps/away/lost_supply_base/lost_supply_base.dmm
#	maps/away/meatstation/meatstation.dmm
#	maps/away/mining/mining-asteroid.dmm
#	maps/away/scavver_gantry/scavver_gantry-1.dmm
#	maps/away/scavver_gantry/scavver_gantry-2.dmm
#	maps/away/skrellscoutship/skrellscoutship_shuttles.dm
#	maps/away/slavers_base/slavers_base.dmm
#	maps/away/smugglers/smugglers.dm
#	maps/away/voxship/voxship.dm
#	maps/away/yacht/yacht.dmm
#	maps/away_sites_testing/away_sites_testing.dm
#	maps/event/sfv_arbiter/sfv_arbiter.dmm
#	maps/example/example-1.dmm
#	maps/example/example-2.dmm
#	maps/example/example-3.dmm
#	maps/mapsystem/maps.dm
#	maps/random_ruins/exoplanet_ruins/crashed_probe/crashed_probe.dmm
#	maps/random_ruins/exoplanet_ruins/oldlab2/oldlab2.dmm
#	maps/torch/datums/supplypacks/security.dm
#	maps/torch/items/clothing/solgov-accessory.dm
#	maps/torch/items/rigs.dm
#	maps/torch/structures/memorabilia.dm
#	maps/torch/torch.dm
#	maps/torch/torch_procs.dm
#	maps/torch/torch_shuttles.dm
#	maps/torch/torch_simplemobs.dm
#	packs/deepmaint/deepmaint_rooms/core/telecoms.dmm
#	packs/deepmaint/deepmaint_rooms/normal/engine_room.dmm
#	packs/deepmaint/deepmaint_rooms/normal/medical.dmm
#	test/check-paths.sh
#	test/check_icon_state_limit.py
#	test/run-test.sh

Glloyd committed:
Merge pull request #1633 from UristMcStation/baymerge-testing

merges baymerge-testing into master

Glloyd committed:
Merge branch 'master' of https://github.com/UristMcStation/UristMcStation into baymerge-testing

# Conflicts:
#	code/datums/uplink/ammunition.dm
#	code/datums/uplink/devices and tools.dm
#	code/game/objects/items/weapons/tools/screwdriver.dm
#	maps/away/lar_maria/lar_maria-1.dmm
#	maps/away/lar_maria/lar_maria-2.dmm

ggermanicus commented:
> The moving_quickly thing in mob_movement.dm. What was changed here in the last merge @grant?

I'm guessing it was something I did to get rid of sprinting (since our default movespeed is way faster than Bay's and it was tied to Athletics)

Glloyd commented:
that makes sense, i'll have to take a look to make sure it's still excised. also i pinged some random Grant, oops

grant commented:
Hey, hope you are doing OK. I noticed all the swear words in your tags. Don't worry about pinging me!

Glloyd committed:
addressing the effectpocalypse

Glloyd commented:
Thanks for the concern Other Grant, we'll get through it hopefully.

As for the merge, a cool 18000+ lines later and the effect fuckening is addressed, hopefully. That's the biggest thing for the maps, so once we get this compiling, fixing the maps should go relatively smoothly.

scrdest commented:
I'll lend a hand whenever I can. IIRC the zombo changes were backports from future!Bay, so it might be fine.

**Process improvement idea**: we could run the compiler in a terminal (as opposed to the DreamMaker window) with error limit disabled and post a comment with all the stuff to fix here. 

Will make it easier to track progress (-> less demoralizing) and split the efforts. Some stuff will only pop up when other errors are fixed, but should be manageable. Hell, I could look into setting up a GitHub Action to run this if there's interest.

Laniakeaaa committed:
.dmi pass on most broken icons so far.

Laniakeaaa committed:
Unfucks all the buttons, nice.

Commit:
Whole bunch of compile fixes. Most notably: fixed all the preprocessor-level issues and all the attackby refactor bits. Random grab-bag of fixes everywhere else.

Glloyd committed:
Merge pull request #1632 from Laniakeaaa/monke

Monkeys have a chance to activate grenades if they pick one up.

Glloyd committed:
Merge pull request #1629 from Laniakeaaa/baklava

Adds Sec-Tech Vendor Premium Item

Glloyd committed:
Merge pull request #1630 from Laniakeaaa/binarykey

Binary Translator Key now has correct desc + lower tc slightly.

Glloyd committed:
Merge pull request #1631 from Laniakeaaa/traitoruplinkv100

Traitor Uplink - Maneater Seeds, Moonshine, Description Pass, Name Adjustments, etc.

Laniakeaaa committed:
Fixing broken type-paths

Laniakeaaa committed:
datum/species to singleton/species, quick vox fixes.

Glloyd committed:
Merge pull request #18 from Laniakeaaa/dmi-icon-pass

.dmi pass on most broken icons so far.

Glloyd committed:
Merge pull request #20 from Laniakeaaa/typepath-fixing

Unidentified Type Path Fixes.

Laniakeaaa committed:
Reverts Vox organ changes.

Glloyd committed:
Merge pull request #19 from Glloyd/scrdest/baymerge-25-fixes-1

Whole bunch of compile fixes

Laniakeaaa committed:
fixed xeno to xenos

Laniakeaaa committed:
Merge branch 'baymerge-2025' into species-quickficks

Laniakeaaa committed:
Fixes for Baymerge again

Glloyd committed:
Merge pull request #21 from Laniakeaaa/species-quickficks

Datum/Species moved to Singleton/Species, Vox Organ adjusted for change.

Glloyd committed:
Merge pull request #22 from Laniakeaaa/baymerge-fixes-2

Fixes for Baymerge again

Glloyd committed:
Merge branch 'dev' of https://github.com/Baystation12/Baystation12 into baymerge-testing

# Conflicts:
#	.github/workflows/test.yml
#	baystation12.dme
#	code/__defines/admin.dm
#	code/__defines/mobs.dm
#	code/__defines/species.dm
#	code/_helpers/icons.dm
#	code/_helpers/medical_scans.dm
#	code/_helpers/unsorted.dm
#	code/game/atoms.dm
#	code/modules/ai/ai_holder_pathfinding.dm
#	code/modules/client/preference_setup/antagonism/01_candidacy.dm
#	code/modules/clothing/under/skirts.dm
#	code/modules/events/meteors.dm
#	code/modules/integrated_electronics/subtypes/smart.dm
#	code/modules/mob/living/bot/bot.dm
#	code/modules/mob/living/bot/farmbot.dm
#	code/modules/mob/new_player/new_player.dm
#	code/modules/modular_computers/hardware/scanners/scanner_medical.dm
#	code/modules/overmap/exoplanets/_exoplanet.dm
#	code/modules/overmap/ships/ship.dm
#	code/modules/species/species.dm
#	code/modules/species/station/lizard_subspecies.dm
#	config/example/config.txt
#	maps/torch/torch_simplemobs.dm

Glloyd committed:
Merge branch 'baymerge-2025' of https://github.com/Glloyd/UristMcStation into baymerge-testing

# Conflicts:
#	code/__defines/research.dm

Glloyd committed:
purges T_BOARD, cleans up some merge issues, start on clearing dreamchecker issues

Glloyd committed:
Merge branch 'master' of https://github.com/UristMcStation/UristMcStation into baymerge-testing

# Conflicts:
#	code/datums/uplink/badassery.dm
#	code/game/machinery/vending/security.dm
#	code/game/objects/items/devices/uplink_random_lists.dm
#	code/modules/hydroponics/seed_datums.dm
#	code/modules/hydroponics/seed_packets.dm
#	code/modules/species/station/monkey.dm

Glloyd committed:
fixes vox stack issues

Glloyd commented:
Okay, that brings us fully up to date with both Bay and Urist-master, and fixes some more issues. I won't have a chance to look at this until Monday or Tuesday, but when I get back to this I'm going to work on clearing out dreamchecker and testing errors.

Laniakeaaa committed:
Sharpened Floor + Wep Construction

Commit:
chore: Generic Baymerge fixes part second.

Key changes:
- The ported radio jammer got renamed to radio_jammer_urist because it was conflicting with Bay's new one. Ours can be improvised, so I didn't want to delete, at least not yet.
- Similarly, Urist weather controller got renamed since Bay now has Polaris weather.
- Conference etc. changes to Holopads got nuked - the code was too much of a mess of two codebases. Safer to readd than to break all of the logic with stale mergecode.
- For Cultures, made best guess how to remap things where Bay changed it.

Commit:
fix: Fixed Candidacy warnings and reverted skill stuff that slipped into mechs.

Commit:
fix: More skill cockroaches reee

Glloyd committed:
Merge pull request #23 from Glloyd/baymerge-25-fixes-2

chore: Generic Baymerge fixes part second.

Glloyd committed:
cleans up dna scanner code a bit and clears some dreamchecker errors

Glloyd committed:
further dna_modifier fixing, restores autowrapping disposal chute

Glloyd committed:
fixes dreamchecker issues, probably

Glloyd committed:
huge amount of map fixes, should fix all the nerva compile errors. glloydstation and aways might still have some issues.

Glloyd commented:
dreamchecker is fixed, and the compile errors on nerva are fixed. I've fixed a lot of the broken paths in other maps as well, but there'll still be some outstanding issues that will need to be cleaned up. Next up is fixing tests, but I don't think I'll have a chance to work on this until the weekend, so anyone else is welcome to pick up where I left off before then.

Glloyd committed:
fixes some culture issues

Glloyd commented:
I'm definitely going to need a hand figuring out why atoms aren't initializing in the nerva test.

Laniakeaaa committed:
map fix pass.

Glloyd committed:
Merge pull request #24 from Laniakeaaa/map-fix-pass

Merge Map Fixing - Aways, Exoplanets, etc.

Naxodile committed:
fixes twohanding infinite loop

Commit:
fix: Fixes the cursed gun atom init bug (sneaky endless loop between two calls)
fix: Removes some other now-unnecessary icon calls in guncode.
fix: Fixes up bad areas in example_unit_testing.dm
fix: Fixes up bad alist init in 516 (that one's on Bay)
fix: Resets Example Z2 to Bay. Pain to maintain.

Glloyd committed:
Merge pull request #25 from Glloyd/fix/cursed-gun-bug

15 Apr 25 baymerge fixes

Laniakeaaa committed:
Nerva + Map Adjustments

Glloyd committed:
Merge pull request #26 from Laniakeaaa/baymerge-map-crud

Baymerge Map Cleanup - Nerva + Some Away Fixes.

Laniakeaaa committed:
Restores bluespace river from icon issues.

Laniakeaaa committed:
icon defuckening

Glloyd committed:
Merge pull request #27 from Laniakeaaa/restoresbluespaceriver

Restores Bluespace River's Icons back.

Glloyd committed:
Merge pull request #28 from Laniakeaaa/fixing-more-icons-hell

Iconmines... for Aways.

Laniakeaaa committed:
non-defined glass has a temp icon

Glloyd committed:
Merge pull request #1611 from Laniakeaaa/AdditionalGlloydCrud

Additional Glloydstation Fixing

Glloyd committed:
Merge pull request #1635 from Laniakeaaa/quicktweaks

Sharpened Floor + Wep Construction Adjustment

Glloyd committed:
Merge pull request #30 from Laniakeaaa/glassfix

Glass Shard Icons for Mapping

Naxodile committed:
re-add body_hair caching

haunted dead code...

Glloyd committed:
Merge pull request #31 from Naxodile/baymerge-baymerge

re-add body_hair caching

Naxodile committed:
New IPC default culture fix

Naxodile committed:
Fix baymerge construction states

Bioprinters can be printed, so can our holopads

Naxodile committed:
Fixes baymerge language conflicts

Zombies should only speak zombie, and the letter drought is dire

Laniakeaaa committed:
iconmines pt2 + new away + bluespace

Naxodile committed:
Reverts IPC culture back to normal

Glloyd committed:
Merge pull request #32 from Naxodile/baymerge-baymerge

Fixes remaining Nerva unit test issues

Glloyd committed:
Merge pull request #33 from Laniakeaaa/iconmines2

Iconmines Part 2 + Bluespace Interlude + Abandoned Hotel

Glloyd committed:
removes unused or deprecated code that was causing some code tests to fail

Glloyd committed:
Merge branch 'baymerge-2025' of https://github.com/Glloyd/UristMcStation into baymerge-testing

Glloyd committed:
Merge branch 'master' of https://github.com/UristMcStation/UristMcStation into baymerge-testing

# Conflicts:
#	code/game/objects/items/stacks/tiles/tile_types.dm
#	code/modules/urist/modules/shipbattles/shipweapons/weapon_construction.dm
#	maps/glloydstation/Glloydstation2-1.dmm
#	maps/glloydstation/Glloydstation2-5.dmm
#	maps/glloydstation/Glloydstation2-7.dmm
#	maps/glloydstation/glloydstation_define.dm

Glloyd committed:
awaymap test fixes

Glloyd committed:
makes glloydstation compile, fixes some misc broken paths on other maps

Glloyd committed:
fixes map testing issues on awaymaps, glloydstation, and templates

Laniakeaaa committed:
iconmines3

Laniakeaaa committed:
forgot to bluespace...

Laniakeaaa committed:
recovers example stuff

Glloyd committed:
Merge pull request #34 from Laniakeaaa/glloydstation-iconmines

Icon Mines Part 3 - Glloyd + Random Outliers on other maps.

Glloyd committed:
Merge pull request #36 from Laniakeaaa/recovers-old-example

Recovers old Example Changes.

Glloyd committed:
Merge pull request #35 from Laniakeaaa/actually-makes-bluespace-work

Bluespace Interlude - Additional dependancies.

Glloyd committed:
minor unit testing fixes

Glloyd committed:
Merge branch 'baymerge-2025' of https://github.com/Glloyd/UristMcStation into baymerge-testing

Glloyd committed:
allows skrell scout ship to spawn, with a reduced pick weight.

Glloyd committed:
what i get for not using VSC is missing quotation marks apparently

Glloyd committed:
resolved glloydstation_define.dm being duplicated in the dme, moves bluespace interlude to nerva_procs.dm

Glloyd committed:
moves a few other nerva procs from the define to nerva_procs.dm

Glloyd committed:
missing circuitboard for the awaymap variant of the longrange telecomms

Laniakeaaa committed:
fixing random broke wires

Laniakeaaa committed:
changes

Glloyd committed:
Merge pull request #37 from Laniakeaaa/wire-fixing-test

Fixing Broken Wires

Commit:
fix: Map template fixes, mostly Skrell-adjacent fsr

Commit:
fix: Stupid oversight on Skrell lights

Commit:
fix: More Biodome fixes, humanizes a syndie in Blacksite

Commit:
fix: Annoying uniqueness-on-desc for access fixed. De-airlessifies shuttle template (area should be airless inherently).

Laniakeaaa committed:
example fix

Glloyd committed:
Merge pull request #39 from Laniakeaaa/fixes-example-again

Actually fixes example-2

Glloyd committed:
Merge pull request #38 from Laniakeaaa/grant-movement

Adjusts: Run, Walk, Creep Movements.

Laniakeaaa committed:
Gunpocalypse-Pt1

Glloyd committed:
resolves some awaymap issues

Glloyd committed:
allows the custom email preference setting to work properly, and fixes a runtime with skrell scoutship

Glloyd committed:
replaces the old inert cryopods with an actual object

Glloyd committed:
resolves some gun issues

Glloyd committed:
resolves glass icon preview issues

Glloyd committed:
i hate the new map editor

Glloyd committed:
resolves template testing issues

Glloyd committed:
resolves overlay related code test issues

Glloyd committed:
clears indentation error, fixes other overlay issues

Glloyd committed:
updates check-paths.sh

Glloyd committed:
resolves .len tests

Glloyd committed:
resolves code tests probably

Glloyd committed:
missed a couple isturf()s

Laniakeaaa committed:
Final Fixes.

Laniakeaaa committed:
restores old bolt sound for now

Glloyd committed:
Merge pull request #41 from Laniakeaaa/urist-maintenance-gunpocalypse

Urist Gun Maintaining: Updating Weapons to Current Baymerge Functionality

Glloyd commented:
I'm merging this into baymerge-testing. Tests are passing, although there's still a decent amount of work to do here, but that's what this branch is for. This way we can test PRs properly on the main repo and view diffs more easily, outside this fuckhuge PR.
